src/digisos/skjema/familie/sivilstatus/FamilieTypes.ts

```diff
@@ -47,13 +47,15 @@ export const initialFamilieStatus: Familie = {
     forsorgerplikt: initialForsorgerPlikt
 };
 
-export const initialPerson: Person = {
-    navn: {
-        fornavn: "",
-        mellomnavn: "",
-        etternavn: "",
-        fulltNavn: ""
-    },
-    fodselsdato: null,
-    personnummer: null
+export const lagInitialPerson = ():Person => {
```

Comment:
Denne klinget ikke så godt for meg. Hva med ```createInitialPerson```?

Reply:
Sant sant. Den var alt på to språk, og jeg valgte ikke med omhu.